front/app/containers/Admin/initiatives/settings/index.tsx

```diff
@@ -169,6 +195,12 @@ const InitiativesSettingsPage = () => {
         await toggleProposals({ enabled: newProposalsNavbarItemEnabled });
       }
 
+      if (proposalsPageBodyChanged) {
+        await updatePage(proposalsPage.id, {
```

Comment:
Might be slightly faster to group the calls and await them all (ie by filling in an array with promises), so they can be all sent at the same time instead of chained and awaiting the previous one to b done to start the next.